backend/src/api/cycle.js

```diff
@@ -3,14 +3,12 @@ const { Cycle } = require('../models')
 const { directorsOnly, errorWrap } = require('../middleware')
 const router = express.Router()
 
-// Get all cycles (previous and current)
-// TODO: when authentication server is integrated, only show the cycles that
-// belong to the caller of this endpoint.
+// Get all cycles belonging to the same workspace that a User is in (previous and current)
 router.get(
   '/',
   [directorsOnly],
   errorWrap(async (req, res) => {
-    const cycles = await Cycle.find()
+    const cycles = await Cycle.find({ workspaceName: { $in: req._user.workspaceIds } })
```

Comment:
is there a diff between `workspaceName` and `workspaceIds`? if not, I think it would make more sense if we only used one of them to prevent confusion but this is more a preference/not a blocker

Reply:
There's more stuff to be done so I can address it in a follow-up